ForesightArtifact/ForesightArtifact.cs

```diff
@@ -76,6 +78,12 @@ public void Awake()
 #endif
         }
 
+        private int RaiseChestPrices(On.RoR2.Run.orig_GetDifficultyScaledCost_int orig, Run self, int baseCost)
+        {
+            var origPrice = orig(self, baseCost);
+            return (int)(origPrice * 1.5f);
```

Comment:
Turn into a const at the top please, we might want to make it configurable in the future